scripts/zones/Sacrarium/npcs/_0st.lua

```diff
@@ -11,6 +11,7 @@ local entity = {}
 entity.onTrigger = function(player, npc)
     if player:getZPos() < 50 then
         npc:openDoor()
+        GetNPCByID(16892140):openDoor()
```

Comment:
Should put this NPC into the IDs.lua and reference it that way instead of putting the ID in directly.

Reply:
you bet